tools/src/scie_pants/pants_version.py

```diff
@@ -117,12 +121,22 @@ def determine_tag_version(
         github_api_url = (
             f"https://api.github.com/repos/pantsbuild/pants/git/refs/tags/{urllib.parse.quote(tag)}"
         )
+        github_releases_url = "https://github.com/pantsbuild/pants/releases"
```

Comment:
So we're actually about to switch to a GitHub Release-based approach for any version >=2.

If you're able to sit a smidge, this code might look different when we make that switch.

Reply:
Let me know when that happens and I can revisit.

Reply:
Thanks for waiting! The switch has happened now.

For version strings that look new (after `PANTS_PEX_GITHUB_RELEASE_VERSION`), this code path isn't used, as the function returns early. For older ones, definitely still worth considering improvements'; thanks!

For those new ones, Pants is then downloaded from GitHub directly, with some error handling about it not existing (but feel free to fine-tune if you have ideas!), in https://github.com/pantsbuild/scie-pants/blob/91477f6394e8eed048e093538b4a4c299514e44f/tools/src/scie_pants/install_pants.py#L88-L116